docs/using-auth-data-synchronization.md

```diff
@@ -69,3 +69,57 @@ projects_black_list: ["id1", "id2"]
 # List of data types to synchronize
 "data_types_to_sync": ["projects", "role_assignments"]
 ```
+
+## Full Example using Keystone for Authentication and RBAC for Authorization and synced namespaces
+
+Run k8s-keystone-auth like this:
+
+```
+k8s-keystone-auth \
+--tls-cert-file /etc/kubernetes/pki/apiserver.crt \
+--tls-private-key-file /etc/kubernetes/pki/apiserver.key \
+--keystone-url https://keystone:5000/v3 \
+--sync-config-file /etc/kubernetes/syncconfig.yaml
+```
+
+This way we use it only for authentication. It will print the following in the log:
+
+```
+W0809 15:23:26.718723       1 config.go:70] Argument --keystone-policy-file or --policy-configmap-name missing. Only keystone authentication will work. Use RBAC for authorization.
+```
+
+Create a ClusterRole like this:
+```
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: _member_
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
+```
+
+Where the important bit of information is `_member_`: the string that represents the role of a user in keystone. If your role is called differently in your keystone setup, like `Member` or `member` you have to change this.
```

Comment:
I believe I'll add a mapper in this case keystone_role->k8s_role, but for now it's true.